src/averageAges.js

```diff
@@ -37,7 +40,20 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const filterWom = people.filter(wom => withChildren
+    ? findMomByName(people, wom) : wom.sex === 'f');
+
+  const ageWom = filterWom.map((age) => differenceAge(age));
```

Comment:
Same as line 14, consider renaming the parameter `age` to something like `person`.